analyzers/tests/SonarAnalyzer.UnitTest/TestCases/InsteadOfAny.EntityFramework.cs

```diff
@@ -17,12 +17,11 @@ public class MyDbContext : DbContext
 
     public void GetEntities(MyDbContext dbContext, List<int> ids)
```

Comment:
could you also add the async scenario from the customer repro:

- the async methods with `await`
- maybe with another context named `foo` to ensure we don't rely on the name of the object (e.g. assume all contexts are named `dbContext`, for example)?
```
_ = await foo
            .MyEntities
            .Where(e => ids.Any(i => e.Id == i))
            .ToListAsync();
```